src/MeetingsSDKAdapter.js

```diff
@@ -679,7 +679,7 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
     }
 
     const enableSharingStream = async () => {
-      const [, localShare] = await sdkMeeting.getMediaStreams({sendShare: true});
+      const [localShare] = await sdkMeeting.getMediaStreams({sendShare: true});
```

Comment:
doesn't `getMediaStreams` return an array with 2 media streams? one for video and one for screen share, if you get the first one, wouldn't that be the wrong one?

Reply:
Honestly I didn't know which one in the right one since the naming doesn't correspond with the one from api documentation. I reverted the changes.